src/Types/ContractEvent.cs

```diff
@@ -9,4 +12,31 @@ public sealed record ContractEvent(byte[] Bytes)
     /// Return hex representation.
     /// </summary>
     public string ToHexString() => Convert.ToHexString(this.Bytes).ToLowerInvariant();
+
+    /// <summary>
+    /// Deserialize event from <see cref="schema"/>.
+    /// </summary>
+    /// <param name="schema">Module schema in hexadecimal.</param>
+    /// <param name="contractName">Contract name.</param>
+    /// <returns><see cref="Bytes"/> deserialized.</returns>
+    /// <exception cref="InteropBindingException">Thrown when event wasn't able to be deserialized from schema.</exception>
+    public string? GetDeserializeEvent(
```

Comment:
**Suggestion:** Make signature like `TryParse` and indicate in the name that this can fail (`Try` prefix).

Since this is parsing the bytes in the event, it should probably follow the same style as `TryParse`, if you would like we can also provide a function which throws.

Reply:
Then penalty of throwing the exception is already present since it is thrown from the interops operation. I have had this discussion with some from UFT team when introducing alike in the CCD scan, and we concluded best C# practice was to have an exception thrown. 
I believe in must cases the user would like to have the exception if it fails, instead of a silent swallow with a `Try`.